app.rb

```diff
@@ -93,6 +106,7 @@ def create_a_book
     book = Book.new(title, author)
     @books << book
     p 'Book created sucessfully!'
+    save_books
   end
 
```

Comment:
![image](https://github.com/over-geek/OOP-School-Library/assets/93039359/32811bf5-2bbd-4bd6-8428-6bab956b9d1e)

- [ ] It would be nice if you fix the highlighted link in the above image because right now this link is broken 😨 Kindly fix it or remove it **(if you don't have any dedicated section for this link 🤔)** to make your readme descriptive and professional 👍